run-system-tests/action.yml

```diff
@@ -211,10 +211,47 @@ runs:
         exit_code=0
         echo "$SMART_CONTRACT_OWNER_WALLET_JSON" >> ./config/sc_owner_wallet.json
         echo '{"client_id": "591d6d0d5642bdbc924756ca2647b4b59adce0acd02a487ecc4d1bd4669293a8","client_key": "ac8798bcadd5ce5ea14f4745850eec61aa080e6fedc1d769af95c42e0079ba03ee9710e9dc966f7c5eb9cd6ef9184af51869d5a345b9520e507aa55ba22bf902","keys": [{"public_key": "ac8798bcadd5ce5ea14f4745850eec61aa080e6fedc1d769af95c42e0079ba03ee9710e9dc966f7c5eb9cd6ef9184af51869d5a345b9520e507aa55ba22bf902","private_key": "fbdc0bcc12168588e2def826b7ee9d8b4c6f1fbacf22ee5064b7db49482b8f0d"}],"mnemonics": "economy day fan flower between rebuild valid bid catch bargain vivid hybrid room permit check manage mean twelve damage summer close churn boat either","version": "1.0","date_created": "2021-12-04T17:03:37Z"}' >> ./config/blobber_owner_wallet.json
+
+        if [[ "${{ env.NETWORK_URL }}" == "dev-3.devnet-0chain.net" ]];
```

Comment:
I would override this value as is done for [other](https://github.com/0chain/actions/blob/master/deploy-0chain/action.yml#L167) tests instead of spaghetti coding in a wallet per url

Reply:
@stewartie4 I had a chat about this with anish. The miner nodes are necessary for certain initial steps like magicBlock, so they cannot be overridden. We're better off using one wallet per network